### ISSUE TITLE
OSC'elot module is unresponsive to messages

### ISSUE THREAD

**Issue by jamshark70** [The-Modular-Mind/oscelot#7]
Apologies in advance if I've missed something obvious.

I'm trying to send OSC from Pure Data to OSC'elot.

![pd-to-oscelot](https://user-images.githubusercontent.com/318301/163654642-d8bca0cb-3e52-4142-bb9a-264582cdfd43.png)

1. Pd: Click the "connect... 8881" message box. (May also click "format if" to ensure the right argument format.)
2. OSC'elot: Click a mapping slot, then click on a parameter. ("Activate the first mapping slot ... Click on a parameter...")
3. Pd: Click the toggle switch. ("Touch a control or key on your OSC device.")

And... nothing happens.

I confirm using SuperCollider that the message coming from Pd has path "/button" and arguments 1 (int), 1.0 or 0.0 (float) -- "address ending with /button... two arguments, Id (Integer) and Value(0.0 or 1.0)."

![pd-to-oscelot-sc-test](https://user-images.githubusercontent.com/318301/163654696-ce8b1d6e-026e-4626-ac96-ea3c756415b6.png)

Also, the following in SC obtains no result in OSC'elot.

```
n = NetAddr("127.0.0.1", 8881);

n.sendMsg('/button', 1, 0.0);
```

I've been using OSC for, oh, 15 years or so now, so I'm quite certain my plumbing is correct (and the SC-receipt test shows that other software *is* getting the messages -- so there is no problem with mistaken port numbers or firewall or other)... but I'm missing something, clearly... I've got `/button` and an int ID and a float value... so I'm stumped.

**Comment by The-Modular-Mind** [comment]
Hey there, just to confirm you did turn it on right? the screenshot shows the buttons as orange, should be green. If it's failing to start when you click on it, check the VCV log to see if there's any errors. Let me know if that helps.

**Comment by jamshark70** [comment]
OK, I just encountered one issue -- it seems that a prior instance of OSC'elot didn't release the port. When I created a new instance on port 8881, it refused to enable. I could enable it with a different port. So there's something up with deleting modules, perhaps.

Modules typically have a minimalist interface, sure, so you're not gonna put up detailed troubleshooting messages, but that was confusing.

After that, it's working.

I'll be honest that I hadn't guessed the meaning of orange vs green -- obvious to some, but not to me. What if the indicator were dark (disabled) and light green (enabled)? I'd guess that "light off" vs "light on" would be more intuitive than "red light" vs "green light" to many.

I had also erroneously assumed that the module would be enabled immediately upon creation.
